pyproject.toml

```
@@ -15,6 +15,16 @@ skip = ["ddtrace/vendor/", ".riot", ".ddriot", ".tox", ".ddtox", ".eggs", "build
 skip_glob = [".venv*", "ddtrace/profiling/exporter/pprof_*pb2.py"]
 line_length = 120
 
+[tool.cython-lint]
+max-line-length = 120
+exclude = '''
+(
+  .venv*
```

Can we ignore all directories that start with a period (ex: .ddriot, .munirs_special_venv_tehe). This would also ignore .github, .gitlab, .circleci which shouldn't be a problem (I hope).

```suggestion
  .**/
```

---

🤔 _probably_ would work. this is mostly in-line with the excludes we have for our other tools.